rust/operator-binary/src/druid_controller.rs

```diff
@@ -738,6 +781,37 @@ fn add_hdfs_cm_volume_and_volume_mounts(
     }
 }
 
+fn get_ldap_secret_placeholder_replacement_commands(
+    ldap_settings: &Option<DruidLdapSettings>,
+) -> Vec<String> {
+    let mut commands = Vec::new();
+
+    if let Some(ldap_settings) = ldap_settings {
+        if let Some(credentials) = &ldap_settings.ldap.bind_credentials {
+            let volume_name = credentials.secret_class.clone(); // TODO: get another way?
+
+            let ldap_bind_user = format!("$(cat /stackable/secrets/{volume_name}/user)");
+            let ldap_bind_password = format!("$(cat /stackable/secrets/{volume_name}/password)");
```

Comment:
The user and password can also be retrieved with a helper function: https://github.com/stackabletech/operator-rs/blob/main/src/commons/ldap.rs#L83

The secret class name/volume name should not be needed by the operator

Reply:
No problem :) Thanks for the suggestion. I made the switch, and ran into an overseen edgecase: bind credentials are required to configure LDAP in druid. So an error case was added in addition to your suggested change.

Reply:
I'm happy with the changes! Not sure about the anonymous bit, I saw your question in Slack